package.json

```diff
@@ -33,7 +35,7 @@
 		"how-to/automation-testing/selenium",
 		"how-to/automation-testing/wdio"
 	],
-	"howToCustomFolder": "",
+	"packageCustomFolder": "",

```

Comment:
this feature was here to support custom branches. Is this just to make the naming clearer?